docs/flashwear.md

```diff
@@ -0,0 +1,39 @@
+## Flash wear estimation
+
+- $N$: Number of deployments during the lifetime of a device.
+- $F$: Probability of failure during a deployment.
+- $W$: Maximum number of state transitions, defined by `MENDER_MAX_STATE_DATA_STORE_COUNT`.
+- $B$: Number of writes for a successful deployment.
+- $K$: Total number of writes to the flash.
+
+We write to the flash in two places:
+1. during initialization if there are no authentication keys in the store
```

Comment:
We need to mention that this is an opt-in feature. Maybe:

_during first boot if the user did not provide a callback to get the keys, Mender will autogenerate one and write it into the store_